test/runtests.jl

```diff
@@ -1,4 +1,4 @@
-using Base.Test
+using Test
```

Comment:
Should this be Compat.Test?

Reply:
this looks fine with just "Test", it was suggested directly by the compiler